package.json

```diff
@@ -27,7 +27,7 @@
         "fetch": "npm-run-all --parallel fetch:*",
         "start": "npm-run-all build:sass --parallel watch:*",
         "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
-        "lint": "eslint --ext=.js,.jsx . && markdownlint \"**/*.md\" -i node_modules",
+        "lint": "eslint --ext=.js,.jsx . && markdownlint \"**/*.md\" ",
```

Comment:
How about running lint parallely using `npm-run-all`? we can save a little bit as they lint different files?

Reply:
yes I will try.

Reply:
We should ignore `node_modeules`, why did we remove this?